### ISSUE TITLE
Systemd : Requires/After ?

### ISSUE THREAD

**Issue by Poil** [saltstack-formulas/docker-formula#171]
Hi,

I'm asking myself to add the possiblity for orchestring the systemd-docker-unit.
By example I would like to start the Consul container before starting the Vault and Dnsmasq containers

What do you think about this feature ?

**Comment by noelmcloughlin** [comment]
Kinda related to https://github.com/saltstack-formulas/docker-formula/issues/120 - it would be useful to have this feature for separate projects - assuming you are talking about docker composing.

**Comment by Poil** [comment]
I'm not talking about docker composing, Systemd have this possibility natively, so if using only systemd and service file to start docker we are able to orchestrate the run of the containers

**Comment by noelmcloughlin** [comment]
Yes I understand the suggestion now. You want to manipulate `/usr/lib/systemd/systemd/docker.service` to add Before or Wants clauses based on pillars.

**Comment by Poil** [comment]
yes :)
